app/models/framework_question.rb

```diff
@@ -6,6 +6,7 @@ class FrameworkQuestion < VersionedModel
     checkbox: 'checkbox',
     text: 'text',
     textarea: 'textarea',
+    add_multiple_items: 'add_multiple_items',
```

Comment:
the `add_multiple_items` seems a bit of an odd name: would `multiple_items` work better?

Reply:
Indeed naming this question was tough. Our reasoning for prepending it with add is to ensure that it is communicated as allowing a user to `add` or amend the items number, and felt more explicit. `multiple_items` as a name seemed a bit vague for a type of question. We initially started with `add_another` based on https://moj-design-system.herokuapp.com/components/add-another but we didn't feel that was clear either so went with this as a middle ground.

Reply:
👍